apps/web/next.config.js

```diff
@@ -187,6 +187,11 @@ const nextConfig = {
     optimizePackageImports: ["@calcom/ui"],
     turbo: {},
   },
+  i18n: {
+    defaultLocale: "en",
+    locales: ["en"],
+    localeDetection: false,
+  },
```

Comment:
<img width="319" alt="Screenshot 2025-03-24 at 1 55 12 AM" src="https://github.com/user-attachments/assets/78bdbc24-bdbe-46a2-9e90-95c438d75d96" />

we removed this in #18834

I am not adding `...i18n` because it's redundant